docs/dev/issues.md

```diff
@@ -0,0 +1,137 @@
+3. If there are no similar issues, click the [New issue](https://github.com/SuperDARN/pydarn/issues/new/choose) button (top right)
+4. Choose an issue template, or select *open a blank issue* at the bottom of the page if none of the templates seem appropriate
+5. Fill out the template information, and add a descriptive title
+6. Fill out the side bar information:  
+    - Assignees: Assign yourself to the issue if you are planning to work on it yourself. Leave blank if you are not planning to work on it.
+    - Labels: these are used to categorize and filter issues (examples below). You can have multiple labels. 
+    - Projects: Indicate if this issue is part of a [project](projects.md)
+    - Milestone: Please leave this blank. The DAWG chairs will fill in this information later when a timeline emerges (e.g. working towards a new release)
+    - Linked Pull Request: Leave this blank for now. This can be filled out when a pull request that solves this issue is created (see [pull requests](pull_request.md))
+* Click *Submit new issue*. At this point, everyone who is subscribed to the pydarn repository will receive an email. Depending on your notification settings, you may receive emails when others comment on the issue. Check your user settings to subscribe or unsubscribe to email notifications.
+
+
+## Following The Issue
+
+Please follow the ongoing conversation about your issue, especially if other developers request more information. 
+If you do not respond for a while, the issue will go **stale** and may be closed. DAWG tries to keep issues open and contact users if their issues have gone stale. 
+
+Please keep in mind that DAWG members have priorities other than pydarn, and they live in many different time zones, so be patient while waiting for a response. If nobody has responded in 1-2 weeks, it's fine to ask again.
+
+## Closing Issues
+
+Issues should be closed when:
+
+- A [pull request](PR.md) that addresses the issue is merged
+- The original question raised in an issue has been answered
+
+If new issues emerge during a discussion, open a new issue and discuss it there. This is to avoid scope creep.
+
+## Stale Issues
+
+If an issue goes stale (no progress for over 3 months), pyDARN lead developer may follow this procedure:
+
+1. Ask about the issue's status in the conversation
+2. Email the user who opened the issue
+3. Request that we discuss the issue in the next DAWG meeting (add label *telecon*), or place it in a project to be re-opened later
+* Label the issue **stale** until progress can be made on it again
+* Request to close the issue if none of the above steps elicit any information to keep it open
+* DAWG chairs can close **stale** issues
+
+## Assignees
+
+The Assignees section is used to indicate who (if anyone) is actively working on each issue. This helps to avoid duplicated work, and to identify issues that nobody is currently working on (if you are looking for something to do!)
```

Comment:
```suggestion
The Assignees section is used to indicate who (if anyone) is actively working on each issue. This helps to avoid duplicated work, and to identify issues that nobody is currently working on (if you are looking for something to do!).
```